### ISSUE TITLE
changes for NVDA friendlier ESpeak and enhancements for new users

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#14595]
I evaluated a friend's opinion of espeak. He had used eloquence for many years. After I helped him set the variant to Reed, set inflection to 30, lower the pitch by a small amount, he was quite happy with espeak. He also complained that the reason he still uses Jaws is because jaws has many features NVDA does not have by default. I told him you can find most of those features in NVDA addons. He complained that Jaws is nice because it gives you everything in one package already there. so therefore my suggestions are these. 

1. Instead of the addons manager putting you onto a website and you have to go find the addons, please make it more like how window-eyes did things. Add a web scraper to the addons manager that will go out to both:

https://addons.nvda-project.org/index.en.html

and also:

https://nvda-addons.org/
and

https://github.com/BabbageCom/objEnhancer

scrape those sites for all addons, their descriptions and download links. Put them in a list view with checkboxes. have their descriptions in a read only edit box and put an install button and uninstall buttons beside the edit box. This will make it easier for new users to find and install the addons they want, even installing multiple addons if they choose. You could even filter addons by type of addon, if its a voice addon like vocalizer or an addon that just adds new features you want? If its vocalizer it will let you download the addon but its still up to you to purchase it. 

2. Offer two NVDA download options, NVDA basic, and NVDA advanced or NVDA full packages. NVDA advanced or full would be much larger than the basic NVDA because it would contain all compatible NVDA addons. And NVDA basic would be the small traditional NVDA, the one with no addons by default. The one we currently download. NVDA full would be great for new NVDA users because it would just have everything they need or would want in the future all addons preinstalled. They could hit a keystroke for helpful hints and just use the features they want as they become familiar with them. NVDA full would be great for assistive technology trainers and employers because everything they would ever want and need is right there all addons preinstalled. Want to relabel an object in your work application? The object enhancer addon is already there.

**Comment by amirmahdifard** [comment]
hi,
this is my appinian about this, i respect others appinian too of corce.
first, i don't think that in an issue in nvda project on github, it is really a nice thing to say that jaws screen reader has so much more features than nvda or it is a bettar screen reader: read the below lines to see wy i say this.
1: each project has its own features, has its own bads and goods, comparing them is absolutely doesn't makes sence to me, even if it comes to comparing the jaws and nvda point, i beleave that each project can have a feature that the other project doesn't have it: jaws is a paid screen reader: when i use jaws, i feal that it was on my choise to be a blind person, so i need to pay to a cumpeny so that they can allow me to use a computer, but i'm not a blind by my choise, so wy i need to pay heeps of money just because i'm blind and others can use their computer freely? so, in this kace, i pick nvda's side, because nvda doesn't treet me difrent than normal sited users: nvda  is saying, wy sited people should use computers freely but not blind users, nothing is difrent with them: also if jaws has some features that nvda doesn't have some of them i still choos nvda because, first thing that i really like, i can use nvda just by 27 mb of internet spending, wich is good for me because i live in a poor cuntry like iran, wich means that i can't use so meny of my internet to download something like jaws: 2nd, for such who have week computers like me, nvda runs more faster than jaws, in hdd hard drives and with week ram and cpu, wich jaws never did at least in my kace! 3th, simple interface: i like where your nvda settings are not split to  several peaces in your computer, you can manage it only with in 1 settings dialog. 4th, i don't like jaws script system, where as i like nvda addon system.
2nd, i feal it is really a disrespect to nvda developers when it comes to comparing the both screen readers, as they are developing nvda, and they adding more and more features to it everyday over time, so wy we should blame them just because nvda is just not as old screen reader as jaws! if jaws is a good screen reader in some kaces that nvda is not, keep in mind that jaws didn't get on to that point just in one day, they also spend so meny years to get in to that, and nvda is greatly going forward with this kace, even tho nvda is not getting as much as money as jaws, wich makes nvda's developers work really greate for me!
3th, who forced you to just use nvda and not anything else, both projects are available with their bads and goods, with all of their weekness and strongness, you are free to use any of them.
2: they don't need to do a such thing for downloading addons, addons  are available in the websites everywhere of the internet, you can download anything and install it on your nvda as how as you want.
hope this helps, and guys, correct me if i'm rong.
thanks.

**Comment by joshknnd1982** [comment]
No I am not blaming anyone. Perhaps you misunderstood my message. It would make it easier for new users to include as many free open source gnu general public license addons in a larger NVDA package for those who choose to get the larger package. and to have a startup wizard that will let people setup their voice right after setup or maybe set espeak to the Reed voice, volume 100, pitch 50 speed 50 inflection 30.

**Comment by michaelDCurran** [comment]
* eSpeak: Max is the current default and we don't have any plans to change this. Though the user is always able to configure NVDA to what works best for them.
* NV Access already has plans for an add-ons store built into NVDA. Work is ongoing.
* NV Access will not be providing a build of NVDA with popular add-ons pre-packaged.
